src/components/container/ProviderSearch/ProviderSearch.stories.tsx

```diff
@@ -72,7 +88,7 @@ export const Searching: Story = {
 export const Live: Story = {
 	args: {
 		previewState: "Default",
```

Comment:
Not something you added, but Live has no preview state/data. Please delete this arg